### PR TITLE
EFRS-875-inher-app-owner

### PR DESCRIPTION
**Pull request by omorgunova-pl**

### CODE REVIEW

File: ui/src/app/features/user-list/user-list-facade.ts [hunk 1]
```diff
@@ -82,7 +82,7 @@ export class UserListFacade implements IFacade {
       deleteUser({
         userId: deletion.userToDelete.userId,
         deleterUserId: deletion.deleterUserId,
-        newOwner,
+        newOwner: 'owner',
```
Comment (isobolev) [inline ui/src/app/features/user-list/user-list-facade.ts:85]: `newOwner: 'deleter'` is never used anymore?

Comment (omorgunova-pl) [reply ui/src/app/features/user-list/user-list-facade.ts:85]: @isobolev, yes, now it will be always a global owner

Comment (isobolev) [reply ui/src/app/features/user-list/user-list-facade.ts:85]: then we should remove the replacer parameter at all on BE. For now, let's hardcode owner in the endpoint call itself, and remove passing newOwner prop.

File: ui/src/app/features/user-list/user-list-facade.ts [hunk 2]
```diff
@@ -82,7 +82,6 @@ export class UserListFacade implements IFacade {
       deleteUser({
         userId: deletion.userToDelete.userId,
         deleterUserId: deletion.deleterUserId,
-        newOwner,
```
Comment (isobolev) [inline ui/src/app/features/user-list/user-list-facade.ts:85]: please delete newOwner above in params  deleteUser(deletion: UserDeletion, newOwner?: string): void {

Comment (isobolev) [reply ui/src/app/features/user-list/user-list-facade.ts:85]: check where deleteUser is called and delete newOwner there as well if used

Comment (omorgunova-pl) [reply ui/src/app/features/user-list/user-list-facade.ts:85]: done

**Review by isobolev — APPROVED** [review]

**Pull request merged**